src/client/components/ContentCard/DeliveryInformation.component.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import './DeliveryInformation.styles.css';
+
+export function DeliveryInformation({ sampleAddress }) {
+  return (
+    <div className="DeliveryInfo">
+      <label className="addressTitle">DELIVERY INFO:</label>
+      <ul>
+        <li>
+          <label className="addressLabel">Address:</label>
+          <label className="addressData"> {sampleAddress.address}</label>
+        </li>
+        <li>
+          <label className="zipCodeLabel">Zip code:</label>
+          <label className="zipCodeData"> {sampleAddress.zipCode}</label>
+        </li>
+        <li>
+          <label className="cityLabel">City:</label>
+          <label className="cityData"> {sampleAddress.city}</label>
+        </li>
+        <li>
+          <label className="countryLabel">Country:</label>
+          <label className="countryData"> {sampleAddress.country}</label>
+        </li>
+      </ul>
+    </div>
+  );
+}
+
+DeliveryInformation.defaultProps = {
+  editMode: false,
+  sampleAddress: {
+    address: 'Enghavevej 80',
```

Comment:
What you are currently doing is that if the user has not provided an address, you are assigning `Enghavevej 80` as the address, even if they have fx. entered Sweden/Belgium/Japan/Iran/any other country. The same applied to city, zipCode and country. It will be better to have fx. default address as the empty string.